lib/assessment.rb

```diff
@@ -43,6 +47,8 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    file.open(filename) do |file|
+      file.reduce(&:+).gsub(/"\n"/, " ")
```

Comment:
Your order is backwards - you were supposed to do the gsub before concatenating. Also, don't forget to string off any whitespace at the end.